evap/results/templates/results_evaluation_detail.html

```diff
@@ -33,170 +35,188 @@ <h3 class="me-auto">{{ evaluation.full_name }} ({{ evaluation.course.semester.na
-                    title="{% trans 'Shows all results available for you.' %}">
-                    {% trans 'Full' %}
+                    <a href="{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view=full"
+                       role="button"
+                       class="btn btn-sm btn-light{% if view == 'full' %} active{% endif %}"
+                       data-bs-toggle="tooltip"
+                       data-bs-placement="bottom"
+                       title="{% trans 'Shows all results available for you.' %}">
+                        {% trans 'Full' %}
                     </a>
                     {% if not evaluation.can_publish_rating_results %}
-                        <button type="button" disabled class="btn btn-sm btn-light" data-bs-toggle="tooltip" data-bs-placement="bottom"
-                            title="{% blocktrans %}The results of this evaluation have not been published because it didn't get enough votes.{% endblocktrans %}">
+                        <button type="button"
+                                disabled
+                                class="btn btn-sm btn-light"
+                                data-bs-toggle="tooltip"
+                                data-bs-placement="bottom"
+                                title="{% blocktrans %}The results of this evaluation have not been published because it didn't get enough votes.{% endblocktrans %}">
                             {% if evaluation.course.is_private %}
                                 {% trans 'Participant' %}
                             {% else %}
                                 {% trans 'Public' %}
                             {% endif %}
                         </button>
                     {% else %}
-                        <a href="{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view=public" role="button" class="btn btn-sm btn-light{% if view == 'public' %} active{% endif %}" data-bs-toggle="tooltip" data-bs-placement="bottom"
-                           title="
-                        {% if evaluation.course.is_private %}
-                            {% trans 'Shows results available for the participants.' %}">
-                                {% trans 'Participant' %}
+                        <a href="{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view=public"
+                           role="button"
+                           class="btn btn-sm btn-light{% if view == 'public' %} active{% endif %}"
+                           data-bs-toggle="tooltip"
+                           data-bs-placement="bottom"
+                           title=" {% if evaluation.course.is_private %} {% trans 'Shows results available for the participants.' %}">
+                            {% trans 'Participant' %}
                         {% else %}
```

Comment:
djlint got mightily confused here, the indentation is wrong in the rest of the file. To be fair, this code here confused me too.